lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -32,7 +32,10 @@ def __init__(self, err, code):
 
     @property
     def faultString(self):
-        return self.err.getTraceback()
+        try:
+            return self.err.getTraceback()
+        except AttributeError:
+            return self.err
```

Comment:
Does faultString need to be a string, maybe this should be `return str(self.err)`